ios/Classes/AblyInstanceStore.h

```diff
@@ -0,0 +1,32 @@
+@import Foundation;
+@import Flutter;
+
+@class ARTRest;
+@class ARTRealtime;
+#import "AblyFlutterClientOptions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface AblyInstanceStore : NSObject
+
++ (instancetype)sharedInstance;
+
+-(NSNumber *) getNextHandle;
+
+-(void)setRest:(ARTRest *const)rest with:(NSNumber *const)handle;
+
+-(nullable ARTRest *)restFrom:(NSNumber *)handle;
+
+-(void)setRealtime:(ARTRealtime *const)realtime with:(NSNumber *const)handle;
+
+-(nullable ARTRealtime *)realtimeFrom:(NSNumber *)handle;
+
+-(NSNumber *)setPaginatedResult:(ARTPaginatedResult *const)result handle:(nullable NSNumber *)handle;
+
+-(ARTPaginatedResult *) getPaginatedResult:(NSNumber *const) handle;
+
+-(void)reset;
+
+@end
+
+NS_ASSUME_NONNULL_END
```

Comment:
no newline at the end

Reply:
I've done it in 479e1c91b45ff1300b6cfbd875b5cec2a19e7af4, but I wonder what use it brings here.

Reply:
https://unix.stackexchange.com/a/18789